tools/nanopolish/.shed.yml

```diff
@@ -0,0 +1,10 @@
+name: nanopolish
+owner: bgruening
+description: Nanopolish software package for signal-level analysis of Oxford Nanopore sequencing data. 
+homepage_url: https://github.com/jts/nanopolish
+long_description: |
```

Comment:
this one should be a suite enabled file, like this one: https://github.com/galaxyproject/tools-iuc/blob/643293a896a2ccfac10fe995f48c7f01c1a89a7f/tools/biom_format/.shed.yml

This will be in preparation of multiple tools.